File-Storage-Security/Deployment/aws-python-deploy-to-all-existing-s3/deploy.py

```diff
@@ -180,25 +174,23 @@ def deploy_storage(kms_arn, region, bucket_name):
             S3_Encryption,
         ],
         Capabilities=["CAPABILITY_IAM"],
-    )  
+    )
     cft_waiter = cft_client.get_waiter("stack_create_complete")
     cft_waiter.wait(StackName="C1-FSS-Storage-" + cfbucketname)
     res = cft_client.describe_stacks(StackName="C1-FSS-Storage-" + cfbucketname)
     storage_stack = res["Stacks"][0]["Outputs"][2]["OutputValue"]
-    #gather scanner stack id
-    id_call = http.request('GET', stacks_api_url+"stacks",fields={"limit": "100", "type": "scanner"}, headers = {'Authorization': 'ApiKey ' + ws_api, 'Api-Version': 'v1'})
-    try:
-        id_resp = json.loads(id_call.data.decode('utf-8'))['stacks']
-    except json.decoder.JSONDecodeError:
-        time.sleep(1)
-        id_resp = json.loads(id_call.data.decode('utf-8'))['stacks']
+    # gather scanner stack id
```

Comment:
Why remove `try` and `except`?

Reply:
No longer causing an error where its needed

Reply:
Happy to add the try excepts back in